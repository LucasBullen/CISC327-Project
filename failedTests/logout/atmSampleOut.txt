2,6c2,5
< Logged into agent mode.
< Enter account number to create.
< Incorrect account number format (8 digits, no leading zeros). Create cancelled.
< Unrecognized command. Remember all commands must be in lowercase.
< Unrecognized command. Remember all commands must be in lowercase.
---
> Logged into atm mode.
> Cannot access create feature in atm mode.
> Enter amount (in cents) to deposit.
> Deposit successful.
8c7
< Enter amount to withdraw.
---
> Enter amount (in cents) to withdraw.
10,13c9,10
< Enter account number to delete.
< Incorrect account number format (8 digits, no leading zeros). Delete cancelled.
< Unrecognized command. Remember all commands must be in lowercase.
< Unrecognized command. Remember all commands must be in lowercase.
---
> Cannot access delete feature in atm mode.
> Logged out.
