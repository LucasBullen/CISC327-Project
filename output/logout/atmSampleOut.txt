Select mode: atm or agent.
Logged into agent mode.
Enter account number to create.
Incorrect account number format (8 digits, no leading zeros). Create cancelled.
Unrecognized command. Remember all commands must be in lowercase.
Unrecognized command. Remember all commands must be in lowercase.
Enter account number to withdraw from.
Enter amount (in cents) to withdraw.
Withdraw successful.
Enter account number to delete.
Incorrect account number format (8 digits, no leading zeros). 
Unrecognized command. Remember all commands must be in lowercase.
Unrecognized command. Remember all commands must be in lowercase.
