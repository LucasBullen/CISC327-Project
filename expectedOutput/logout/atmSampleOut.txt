Select mode: atm or agent.
Logged into atm mode.
Cannot access create feature in atm mode.
Enter amount (in cents) to deposit.
Deposit successful.
Enter account number to withdraw from.
Enter amount (in cents) to withdraw.
Withdraw successful.
Cannot access delete feature in atm mode.
Logged out.
